_projects/2014-04-26-acoustic-driven-synthesizer.md

```diff
@@ -1,7 +1,8 @@
 ---
+layout: project
```

Comment:
Good news, we can add this tag to our _config to avoid having to repeat it throughout the collection.... sorry you already did that work! 😳